modules/integration/src/test/scala/scala/cli/integration/SipScalaTests.scala

```diff
@@ -6,32 +6,40 @@ import scala.util.Properties
 
 class SipScalaTests extends ScalaCliSuite {
 
-  def noDirectoriesCommandTest(binaryName: String): Unit =
-    TestInputs.empty.fromRoot { root =>
-      val cliPath = os.Path(TestUtil.cliPath, os.pwd)
+  implicit class BinaryNameOps(binaryName: String) {
+    def prepareBinary(root: os.Path): os.Path = {
+      val cliPath    = os.Path(TestUtil.cliPath, os.pwd)
+      val ext        = if (Properties.isWin) ".exe" else ""
+      val newCliPath = root / s"$binaryName$ext"
+      os.copy(cliPath, newCliPath)
+      os.perms.set(newCliPath, "rwxr-xr-x")
```

Comment:
Should we set `perms` on windows?

Reply:
Yeah, I guess there's no need to do that, although it doesn't seem to have been affecting the tests, really...
Anyhow, should be fixed now.